lib/assets/javascripts/twine.js.coffee

```diff
@@ -283,6 +283,15 @@ Twine.bindingTypes =
         $(node).attr(key, value || null)
       lastValue = newValue
 
+  'bind-prop': (node, context, definition) ->
+    fn = wrapFunctionString(definition, '$context,$root', node)
+    lastValue = {}
+    return refresh: ->
+      newValue = fn.call(node, context, rootContext)
+      for key, value of newValue when lastValue[key] != value
+        $(node).prop(key, value || null)
```

Comment:
Hmm though trying to set a value to `undefined` will actually be equivalent to calling `$(el).prop('checked')` which just returns the property